com.microsoft.mrtk.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -106,7 +106,7 @@ public override bool isHoverActive
                 // When the gaze pinch interactor is already selecting an object, use the default interactor behavior
                 if (hasSelection)
                 {
-                    return base.isHoverActive;
+                    return base.isHoverActive && IsTracked;
```

Comment:
Would we want to add `IsTracked` to `CanSelect` and `CanHover` as well?

Reply:
Polyfill technically makes it unnecessary. If such edge cases arise again, we can add the additional checks too.